src/components/Table/index.js

```diff
@@ -118,7 +118,12 @@ class Table extends React.Component {
 Table.propTypes = {
   children: PropTypes.arrayOf(function(propValue, key) {
     for (let i = 0; i < propValue.length; i++) {
-      if (!includes(propValue[i].type.name, [Row.name, Header.name])) {
+      if (
```

Comment:
Why the switch here?

Reply:
I don't believe you can pass an array of possible values to be included into lodash includes. They need to be executed individually.

Reply:
I'm confused - includes should have worked here? Is it not working?? Am I missing something?

Reply:
It's checking if the propValue[i].type.name is either Row.name or Header.name - why doesn't that work?

Reply:
haha ohhh so i think the args are actually just switched

Reply:
wait no because we have the fp lodash, they shouldn't be switched